cpp/src/copying/split.cpp

```diff
@@ -31,7 +31,7 @@ namespace  {
         std::vector<T> result{};
 
         if(splits.size() == 0 or column_size == 0) {
-            return result;
+            return std::vector<T>{input};
```

Comment:
You can also remove the `result` declaration above.

Reply:
Removed 👍